ocaml/xapi/helpers.ml

```diff
@@ -63,6 +63,19 @@ let get_pool ~__context = List.hd (Db.Pool.get_all ~__context)
 let get_master ~__context =
   Db.Pool.get_master ~__context ~self:(get_pool ~__context)
 
+let get_physical_interfaces ~__context ~bridge =
+  let dbg = Context.string_of_task __context in
+  Net.Bridge.get_physical_interfaces dbg ~name:bridge
+
+let is_connected ~__context ~iface =
+  let dbg = Context.string_of_task __context in
+  Net.Interface.is_connected dbg ~name:iface
+
+let get_management_iface_is_connected ~__context =
+  let ifaces = get_physical_interfaces ~__context ~bridge:
+    (Xapi_inventory.lookup Xapi_inventory._management_interface) in
+  List.exists (fun iface -> is_connected ~__context ~iface) ifaces
```

Comment:
Christian would say: use `|>` :) If you drop the named args from the helper functions, this can be:
```
Xapi_inventory.lookup Xapi_inventory._management_interface
|> get_physical_interfaces ~__context
|> List.exists (is_connected ~__context)
```